model/roles.go

```diff
@@ -1184,14 +1291,22 @@ func validateTemplateUsage(roleManifest *RoleManifest) validation.ErrorList {
 
 	// Iterate over the global templates, extract the used
 	// variables. Report all without a declaration.
+	for _, templateDef := range roleManifest.Configuration.Templates {
+		key := templateDef.Key.(string)
+		template := templateDef.Value.(string)
 
-	for _, template := range roleManifest.Configuration.Templates {
 		varsInTemplate, err := parseTemplate(template)
 		if err != nil {
 			// Ignore bad template, cannot have sensible
 			// variable references
 			continue
 		}
+
+		if len(varsInTemplate) == 0 {
+			allErrs = append(allErrs, validation.Forbidden(key,
```

Comment:
Should probably still say `configuration.templates.<key>`, so that it's easier to find

Reply:
I think that makes it harder to find because I can't copy paste and search.

Reply:
You can just… copy only the part after the dot? Compare if a property is overridden in one job, _and_ it's in the global section.

But whatever.